### DISCUSSION TITLE
Pagination styling in dark mode

### DISCUSSION THREAD

**Discussion by fullyloadedtech** [filamentphp/filament#1972] [Ideas]
The pagination for resource tables looks good in Light mode:
![pagi-light](https://user-images.githubusercontent.com/101905909/159537002-3a54d8cf-0bf4-4e37-a387-ed3b59973da8.png)

However in Dark mode the "|" separators between page #s are distracting:
![pagi-dark](https://user-images.githubusercontent.com/101905909/159537115-feaf7d89-1e05-47d4-9d86-8a1053687e8b.png)

If I keep Light mode however I invert the colors, the page separators are not as prominent as Dark mode (looks better IMO):
![pagi-invert](https://user-images.githubusercontent.com/101905909/159538073-135c190d-25c4-4e7f-8ab0-8c6b38e4284b.png)

Would you please consider more focal attention on the numbers, and less on the separators for pagination in Dark mode?

**Comment by ryangjchandler** [comment]
@danharrin This is a good idea I think.

**Reply by fullyloadedtech** [reply]
I can't even see the separators in the 3rd screenshot on my phone out in the sun so that's too dark. Looks different on different screens.

**Reply by ryangjchandler** [reply]
We definitely can't account for all the different screen types and brightness levels. We can definitely make them lighter though.

**Reply by danharrin** [reply]
Fixed by #2324

<img width="1095" alt="Screenshot 2022-05-08 at 21 31 12" src="https://user-images.githubusercontent.com/41773797/167314708-3c70f4bd-85ac-4236-928c-7d4b09b4115f.png">
